tests/raftstore/test_service.rs

```diff
@@ -448,24 +448,25 @@ fn test_mvcc_resolve_lock_gc_and_delete() {
 }
 
 #[test]
-fn test_raft_and_coprocessor() {
+fn test_raft() {
     let (_cluster, client, _) = must_new_cluster_and_client();
 
     // Raft commands
-    {
-        let (sink, _) = client.raft();
-        sink.send((RaftMessage::new(), Default::default()))
-            .wait()
-            .unwrap();
-
-        let (sink, _) = client.snapshot();
-        sink.send((SnapshotChunk::new(), Default::default()))
-            .wait()
-            .unwrap();
-    }
+    let (sink, _) = client.raft();
+    sink.send((RaftMessage::new(), Default::default()))
```

Comment:
We will not allow region 0 later. So here we need to send an invalid region.

Reply:
It is an **invalid** reigon when we do not allow region 0.

Reply:
@siddontang The grpc service won't return any errors no matter what you send. It just redirect the messages to raftstore.